packages/playground/cli/src/blueprints-v1/blueprints-v1-handler.ts

```diff
@@ -142,7 +142,8 @@ export class BlueprintsV1Handler {
 			mountsAfterWpInstall,
 			wordPressZip: wordPressZip && (await wordPressZip!.arrayBuffer()),
 			sqliteIntegrationPluginZip:
-				await sqliteIntegrationPluginZip!.arrayBuffer(),
+				sqliteIntegrationPluginZip &&
+				(await sqliteIntegrationPluginZip!.arrayBuffer()),
```

Comment:
Would this also do the trick?

```suggestion
sqliteIntegrationPluginZip:
				await sqliteIntegrationPluginZip?.arrayBuffer(),
```